Homework_7_Java_Arrays_Loops/JavaArraysAndLoops/src/WaterIntakeTracker.java

```diff
@@ -0,0 +1,21 @@
+public class WaterIntakeTracker {
+    public static void main(String[] args) {
+
+        int[] WaterIntake = {3, 1, 4, 2, 0, 5, 2};
+        int sum = 0;
+        int i = 0;
+
+        while (i < WaterIntake.length) {
+
+            sum += WaterIntake[i];
+            i++;
+        }
+
+        System.out.println("Total number of glasses of water drunk in a week: " + sum);

```

Comment:
Please revisit the task description, there's one important thing that was missed here